sdk/tables/data-tables/samples/typescript/package.json

```diff
@@ -29,12 +29,13 @@
   "homepage": "https://github.com/Azure/azure-sdk-for-js#readme",
   "sideEffects": false,
   "dependencies": {
-    "@azure/data-tables": "../..",
+    "@azure/data-tables": "file:../../projects/azure-sdk-for-js/sdk/tables/data-tables/azure-data-tables-1.0.0-preview.1.tgz",
```

Comment:
Unsure if this change is right.  I'm not sure what would be producing that `.tgz`, and I think given the changes we made earlier today it would be `azure-data-tables-1.0.0-beta.1.tgz`?

Reply:
Good catch this needs to be updated to “latest”